packages/cli-platform-apple/src/tools/prompts.ts

```diff
@@ -85,10 +85,16 @@ export async function promptForDeviceToTailLogs(
     type: 'select',
     name: 'udid',
     message: `Select ${platformReadableName} simulators to tail logs from`,
-    choices: simulators.map((simulator) => ({
-      title: simulator.name,
-      value: simulator.udid,
-    })),
+    choices: simulators.map((simulator) => {
+      const version = simulator.version
+        ? ` (${simulator.version.match(/^(\d+\.\d+)/)?.[1]})`
```

Comment:
One more nit: maybe let's have a reusable utility to get formattedSimulatorName or something like this so we can reuse this regex and logic in other places and prompts